src/wdio/services/AxeService.js

```diff
@@ -9,20 +9,21 @@ let axeCoreSrc;
 */
 export default class AxeService {
   // eslint-disable-next-line class-methods-use-this
-  before(config) {
-    const axeConfig = {
-      inject: true, // True if axeCore script should be injected on each test page.
-      ...(config.axe || {}),
-    };
-
+  before() {
     browser.addCommand('axe', (options = {}) => {
       // Conditionally inject axe. This allows consumers to inject it themselves
       // in the test examples which would slightly speed up test runs.
-      if (axeConfig.inject) {
+      const axeConfig = browser.options.axe;
```

Comment:
we should default inject to true, change this to 
```js
const axeConfig = {
  inject: true,
  ...(browser.options.axe || {}),
};
```

Reply:
Agreed. Most will expect this to be true.